source/features/release-download-count.css

```diff
@@ -1,13 +1,15 @@
-/* Improve spacing on mobile. "*" doesn't work */
+/* Simulates fixed columns. "*" doesn't work */
 .rgh-release-download-count > :is(span, small) {
-	flex: 0 0 auto !important;
+	flex-grow: 0 !important;
+	min-width: 6em;
 }
 
-/* Simulates fixed columns: Size  */
-.rgh-release-download-count .text-sm-left {
-	min-width: 5em;
-}
-/* Simulates fixed columns: Date */
-.rgh-release-download-count .text-right {
-	min-width: 6em;
```

Comment:
Make sure this cause a regression (you can blame it to find it)

Reply:
This is included above (`.rgh-release-download-count > :is(span, small)` also selects this)

Reply:
Ah all good, I was referring to this one, but we haven't fixed it yet:

- https://github.com/refined-github/refined-github/issues/6150

Does this PR fix it by chance?

Reply:
Nope, should we fix it here or in a new PR?

Reply:
Here would be ok since this PR is about alignment too, if you'd like